tests/help_test.toit

```diff
@@ -129,8 +129,8 @@ test_combination:
 
     Options:
```

Comment:
Note: the examples (line 600+) stay the same.

I could try to change those too, but it's a bit harder, and maybe the user wants to show different cases.